src/cljs/ataru/hakija/application_hakukohde_component.cljs

```diff
@@ -210,20 +212,34 @@
    [:h2 @(subscribe [:application/hakukohteet-header])]
    [scroll-to-anchor field-descriptor]])
 
+(defn- select-new-hakukohde-row []
+  (when @(subscribe [:application/hakukohteet-editable?])
+    ;[:
+    [:div
+     [:div.application__hakukohde-row--search-toggle
+      {:on-click hakukohde-search-toggle-event-handler
+       :class
```

Comment:
Tälle `.application__hakukohde-row--search-toggle` diville pitäis asettaa olikohan `:role "button"`